CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 -  Process exit instead of hang in case of uncaught exception in pooled threads
    | [#425](https://github.com/JetBrains/bazel-bsp/pull/425)
 
+### Fixes 🛠️
```

Comment:
```suggestion
```

it's already there

Reply:
Oops, right 😅 fixed https://github.com/JetBrains/bazel-bsp/pull/381/commits/ca2fce4a972632565b519252b13cbeb77eccdc81